projects/aws/image-builder/builder/constants.go

```diff
@@ -10,6 +10,9 @@ const (
 	DefaultAMIVolumeSize           string = "25"
 	DefaultAMIVolumeType           string = "gp3"
 	DefaultAMIManifestOutput       string = "manifest.json"
+	DefaultBaremetalDiskSizeMb     string = "6656"
```

Comment:
I think we don't need to define these 3 default values. We don't want to send some constraint on the min/max size when upstream does not have this limit
Default disk size is set in upstream image-builder in this file https://github.com/kubernetes-sigs/image-builder/blob/main/images/capi/packer/raw/packer.json.tmpl#L173

I think what we want to do is to patch this line to use user value or an environment variable , adn set this environment variable in our CI CodeBuild job https://github.com/kubernetes-sigs/image-builder/blob/main/images/capi/packer/raw/packer.json.tmpl#L16